sqlglot/tokens.py

```diff
@@ -655,4 +655,9 @@ def _scan_var(self):
                 self._advance()
             else:
                 break
-        self._add(self.KEYWORDS.get(self._text.upper(), TokenType.VAR))
+
+        if self._text.upper() in ("LEFT", "RIGHT"):
```

Comment:
are you sure this is needed?

Reply:
i think this is wrong, you just need to add left and right to here

https://github.com/tobymao/sqlglot/blob/main/sqlglot/parser.py#L103